### PR TITLE
Feature/implement design beta

### PR DESCRIPTION
**Pull request by trmdy**
This pull request adds a basic implementation of the beta version of our design. See [here](https://xd.adobe.com/view/3e859615-0703-4c77-6ed5-cf67d31f1621-48b4/) for the full design beta.
The purpose of the pull request is to add the basic framework required for further frontend development.

The primary features of the PR are:
* Add normalize.css to the internal base template.
* Add a template for the header.
* Implement a very basic version of the front page. 
* Add serving of media files, i.e. configure the MEDIA_URL property.
* Add a bunch of helper classes to `main.css`, including some for our primary colours.
* Add a component-template for rendering a thumbnail.
* Add a catch-all/not-found template for routes going to `/internal...` not being otherwise matched.
* Add some helper methods to the User and Summary model for rendering some of the attributes.
* Add configuration for the default location of l10n/i18n files.
* Implement a basic version of the profile page.

Additional features this PR provides:
* Add the `profile_image` attribute as a field to the User admin change interface.
* Add more testing (some not directly relevant to this PR, originally) to the User model.

Missing and TODO:
* The front page is still very incomplete. This page is under more scrutiny in the beta version of the design than the rest. Hence we will wait a bit with implementing the rest.
* Design for the existing summaries and quotes pages are not yet implemented.
* None of the design implementations are pixel-perfect yet. As the design is still in the beta we should avoid doing this until we have frozen specific designs "indefinitely".
* The profile page in particular still renders some fields with placeholder values.

### CODE REVIEW

File: summaries/consts.py [hunk 1]
```diff
@@ -22,4 +22,17 @@
...
+    (SUMMARY_TYPE_OTHER, 'Other'),
+    (SUMMARY_TYPE_KAFE_ANSVARLIG, 'Kafeansvarlig'),
+    (SUMMARY_TYPE_HOVMESTER, 'Hovmester'),
+    (SUMMARY_TYPE_BARSJEF, 'Barsjef'),
+    (SUMMARY_TYPE_SOUSCHEF, 'Souschef'),
+    (SUMMARY_TYPE_ARRANGEMENT, 'Arrangement'),
+    (SUMMARY_TYPE_OKONOMI, 'Økonomi'),
+    (SUMMARY_TYPE_STYRET, 'Styret'),
+    (SUMMARY_TYPE_KSG_IT, 'KSG-IT')
+)
```
Comment (alexaor) [inline summaries/consts.py:35]: Missing Driftsmøtet and DH?

File: users/models.py [hunk 2]
```diff
@@ -64,6 +64,34 @@ def current_commission(self):
...
         return self.ksg_status == KSG_STATUS_TYPES[0][0]
 
+    def get_start_ksg_display(self):
+        """
+        This method returns a "semester-year" display for the start ksg date.
+        Examples:
+            2018-01-01 => V18
+            2014-08-30 => H14
+            2012-12-30 => H12
+        :return:
```
Comment (alexaor) [inline users/models.py]: Returns what?

File: users/models.py [hunk 3]
```diff
@@ -64,6 +64,34 @@ def current_commission(self):
...
+    @property
+    def profile_image_url(self):
+        """
+        profile_image_url is a helper property which returns the url of the user's profile
+        image, if the user has an associated profile image. The reason this exists, is that
+        simply calling `user.profile_image.url` will result in a ValueError if the profile_image
+        is null. We can work around this in templates by doing a bunch of if-conditionals. However,
+        it is more practical to use the `default_if_none` filter, hence this method returns None if the
+        image does not exist.
+        :return:
```
Comment (alexaor) [inline users/models.py]: Returns what?

**Review by alexaor — APPROVED** [review]

File: summaries/consts.py (hunk 1 above)
Comment (trmdy) [reply summaries/consts.py:35]: Yeah, you're right, thanks! This seems to be generally missing, so I'll create a new branch.

**Pull request merged**